README.md

```diff
@@ -12,34 +12,32 @@ Bozza ver. 01.047 – 2020-04-12 &ndash; Scritto con [***Mark Text***](https://m
+  Ancora non abbiamo predisposto un nostro sito web con base dati, dove proporre una raccolta di firme o petizione di sorta.  
+  **Commentare [quella pubblicazione](https://mondiaspora.net/posts/ab05452057880138212d0e36a79e7653) in Diaspora è al momento il modo di supportare il nostro lavoro e segnalarci la vostra approvazoine.** Se vorrete farlo, ci potrete scrivere qualche cosa come:  
   **"Approvo il manifesto *Nuovo Paradigma 1.0* (bozza ver. nn.nnn)"**.  
-  La versione che vi è piaciuta potrete anche copiarvela ovviamente (così come qualunque altra).  
-  **Se avete una "facciata" nel web, potrete segnalare nella stessa il manifesto.**  
-  Spendere 5 o 10 minuti del vostro tempo per aprire un acount gratuito su Diaspora sarà l'occasione di uscire un po' dalle grinfie di Facebook, con la sua censura e la sua connivenza con *Big Data*.
+  La versione che più vi è piaciuta potrete anche copiarla (e lo stesso dicasi per qualunque altra).  
+  **Se disponete di account, nick o una qualunque "facciata web" potrete segnalare l'esitenza del manifesto tramite la stessa.**  
+  Vi ringrazieremo se spenderete 5-10 minuti del vostro tempo per aprire un account gratuito su Diaspora e sarà anche l'occasione di uscire un po' dalle grinfie di Facebook, con la sua censura e la sua connivenza con *Big Data*.
 
 &nbsp;
 
-Almeno durante un primo periodo, questo sarà un lavoro in evoluzione, anche se vorremo marcare il raggiungimento di certe tappe di sviluppo del pensiero e dell'azione (potremmo valutare se effettuare delle *release*).
+Almeno per un primo periodo questo manifesto é da cosiderare come un lavoro in evoluzione, anche se vorremmo marcare il raggiungimento di certe tappe di sviluppo del pensiero e dell'azione entro breve tempo (potremmo valutare se effettuare delle *release*).
 
 **SE AVETE DELLE IDEE CON CUI CONTRIBUIRE, FATEVI AVANTI.**
 
-Abbiamo per ora una sola regola: **niente aggressioni**, **niente stroncature di nessun apporto**. Solo così la creatività di tutti potrà dare i frutti migliori.
+Abbiamo, per ora, una sola chiara regola: **niente aggressioni**, **niente stroncature o commenti di nessun apporto**, affinché la creatività di tutti potrà portare ai frutti migliori.
 
-Saranno benvenuti **giuristi**, **costituzionalisti**, **economisti**, **filosofi**, **scienziati**, **informatici**, **matematici**, **giornalisti**, ***bloggers*** e **pensatori e intellettuali in genere**.
+Saranno benvenuti particolarmente categorie di **giuristi**, **costituzionalisti**, **economisti**, **filosofi**, **scienziati**, **informatici**, **matematici**, **giornalisti**, ***bloggers*** e **pensatori e intellettuali**, così come **traduttori** e **artisti** in genere.
 
-Saranno anche benvenuti **traduttori** e **artisti**.
```

Comment:
**Saranno benvenuti particolarmente categorie di .... **
Forse volevi dire "Saranno categorie particolarmente benvenute: ..."
"particolarmente benvenuti/e" suona già a discriminazione... OK tu _vuoi_ dare un certo tipo di selettività, ma per questo stiamo già facendo una lista di categorie di persone che non si trovano ad ogni angolo di strada.
Inoltre, Io credo che il manifesto sia già così lungo che difficilmente ci si metterebbe "chiunque". Se poi volesse venire il troll a dare fastidio per rallentare i lavori, sai meglio di me che lo farebbe anche se qui pubblicassimo che si ammettono solo geni e santi :innocent: 

Vediamo se posso unire "il meglio" delle due:

Saranno benvenuti **giuristi**, **costituzionalisti**, **economisti**, **filosofi**, **scienziati**, **informatici**, **matematici**, **giornalisti**, ***bloggers*** e in generale **pensatori e intellettuali**, così come **traduttori** e **artisti**.

Reply:
Perfetta, si

Reply:
Vuoi modificare la pull request? (Come preferisci, se ti risulta scomodo e preferisci che apporti io le modifiche frutto di questa discussione me lo dici senza problemi.)